### PR TITLE
feat(HACBS-2471): add rhtap-service-push pipeline

### PR DESCRIPTION
**Pull request by scoheb**
- clone of push-to-external-registry pipeline
- additionally includes call of task update-infra-deployments

### CODE REVIEW

File: catalog/pipeline/rhtap-service-push/0.1/rhtap-service-push.yaml [hunk 1]
```diff
@@ -0,0 +1,282 @@
...
+        - name: containerImageIDs
+          value: $(tasks.create-pyxis-image.results.containerImageIDs)
+        - name: server
+          value: $(params.pyxisServerType)
+        - name: pyxisSecret
+          value: $(params.pyxisSecret)
+      workspaces:
+        - name: data
+          workspace: release-workspace
+    - name: infra-deployments-mr
```
Comment (johnbieren) [inline catalog/pipeline/rhtap-service-push/0.1/rhtap-service-push.yaml:240]: pr not mr

File: catalog/pipeline/rhtap-service-push/0.1/rhtap-service-push.yaml [hunk 2]
```diff
@@ -0,0 +1,282 @@
...
+      taskRef:
+        resolver: "git"
+        params:
+          - name: url
+            value: https://github.com/redhat-appstudio/release-service-bundles.git
+          - name: revision
+            value: main
+          - name: pathInRepo
+            value: catalog/task/cleanup-workspace/0.1/update-infra-deployments.yaml
```
Comment (johnbieren) [inline catalog/pipeline/rhtap-service-push/0.1/rhtap-service-push.yaml:249]: this is wrong

File: catalog/pipeline/rhtap-service-push/0.1/README.md [hunk 3]
```diff
@@ -0,0 +1,25 @@
+# RHTAP Service Push to External Registry Pipeline
+
+Tekton pipeline to push images to an external registry.
```
Comment (johnbieren) [inline catalog/pipeline/rhtap-service-push/0.1/README.md:3]: should probably mention something about the pr so it is clear how it differs from push-to-external-registry

File: catalog/pipeline/rhtap-service-push/0.1/rhtap-service-push.yaml (hunk 1 above)
Comment (scoheb) [reply catalog/pipeline/rhtap-service-push/0.1/rhtap-service-push.yaml:240]: fixed

File: catalog/pipeline/rhtap-service-push/0.1/rhtap-service-push.yaml (hunk 2 above)
Comment (scoheb) [reply catalog/pipeline/rhtap-service-push/0.1/rhtap-service-push.yaml:249]: fixed

File: catalog/pipeline/rhtap-service-push/0.1/README.md (hunk 3 above)
Comment (scoheb) [reply catalog/pipeline/rhtap-service-push/0.1/README.md:3]: updated

File: catalog/pipeline/rhtap-service-push/0.1/README.md [hunk 4]
```diff
@@ -0,0 +1,27 @@
+# RHTAP Service Push to External Registry Pipeline
+
+* Tekton pipeline based on "push-to-external-registry".
+* In addition, it can create a GH pull request in infra-deployments.
```
Comment (johnbieren) [inline catalog/pipeline/rhtap-service-push/0.1/README.md:4]: can or does? Doesn't it always?

**Review by johnbieren — APPROVED** [review]

**Pull request merged**